### PR TITLE
SP-11504: add limit to the 'sar_in' queue

### PR DESCRIPTION
**Pull request by przemyslawfierek-silvair**

### CODE REVIEW

**Comment by michallowasrzechonek-silvair** [comment]
I'd rather fix the problem

**Comment by przemyslawfierek-silvair** [comment]
Are you aware that the solution is the modification of the SAR mechanism (rewrite it) to reduce the number of used FD descriptors? AFAIK all features that we provide within the GW do not use the SAR mechanism at all (correct me if I'm wrong), so there is no business requirement to have this done (now). Let's talk about that on Monday.

**Comment by michallowasrzechonek-silvair** [comment]
If so, then maybe just drop all SAR frames altogether?

**Comment by przemyslawfierek-silvair** [comment]
I've tried and it works so this is an option.

File: mesh/net.c [hunk 1]
```diff
@@ -1959,8 +1960,18 @@ static bool seg_rxed(struct mesh_net *net, bool frnd, uint32_t iv_index,
 		uint16_t len = MAX_SEG_TO_LEN(segN);
 
 		l_debug("RXed (new: %04x %06x size: %d len: %d) %d of %d",
-				seqZero, seq, size, len, segO, segN);
+			seqZero, seq, size, len, segO, segN);
```
Comment (michallowasrzechonek-silvair) [inline mesh/net.c]: Formatting?

**Review by michallowasrzechonek-silvair — APPROVED** [review]

**Comment by michallowasrzechonek-silvair** [comment]
For the record: we've decided *not* to drop all SAR transactions, since they are useful when remote access is used by support users - particularly by https://github.com/homersoft/ceiling

**Pull request merged**